### DISCUSSION TITLE
How to execute individual test locally (for testing the tests)?

### DISCUSSION THREAD

**Discussion by fritzrehde** [uutils/coreutils#5242] [Q&A] [answered]
I think I need to enable some feature, but `cargo test --features timeout <test-name>` gives the error e.g.:
```
error: none of the selected packages contains these features: timeout
```

**Comment by fritzrehde** [comment]
Ok, nevermind, I think I figured it out: 1. Install nightly 2. Run the command I tried from the base directory, not in another subdirectory.

**Comment by tertsdiepraam** [comment, marked as answer]
This should work on stable I believe (from the root dir):
```
cargo test --no-default-features --features timeout -- <test-name>
```
